wordpress-mobile/WordPress-iOS
# Update Tracks to 0.8.0

Upgrades Sentry and cleans up associated code.

To test:
- Download the installable build from this PR onto a test device
- Go to Me > App Settings > Debug, and tap "Send Log Message", then "Send Test Crash". Relaunch the app.
- Go to https://sentry.io/organizations/a8c/discover/results/ and enter `user.email:$your_email`, and note that the events show up (might take up to 5 minutes)

PR submission checklist:
